### PR TITLE
Allow svgo plugins to be disabled

### PR DESCRIPTION
**Pull request by tomhughes**

### CODE REVIEW

**Comment by Firefishy** [comment]
The Travis CI build failure seems to be a false positive.

File: lib/image_optim/worker/svgo.rb [hunk 1]
```diff
@@ -4,11 +4,17 @@ class ImageOptim
   class Worker
     # https://github.com/svg/svgo
     class Svgo < Worker
+      DISABLE_PLUGINS_OPTION =
+      option(:disable_plugins, [], Array, 'List of plugins to disable'){ |v| v }
```
Comment (toy) [inline lib/image_optim/worker/svgo.rb]: No need to provide `Array` as an argument as type will be guessed from `[]`.
Better to use `Array(v).map(&:to_s)` in the proc to have no problems with single values and arrays with non string values.

**Comment by toy** [comment]
Great!
It will be fair also to add `enable_plugins` option.
And please add entry to `CHANGELOG.markdown`.

@Firefishy Yep, a bug in rubocop, resolved on master

**Comment by tomhughes** [comment]
Hopefully that should be all your comments addressed now.

**Comment by toy** [comment]
Thanks, merged 8d534db21c452b4ff2884c24f7069abb9f30f494

**Pull request closed without merging**